blog/releases/announcing-knative-v0-23-release.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Version v0.23 release"
```

Comment:
I think the idea was that it's just v0.2x rather than "Version v0.2x" 🙂 sorry. If you don't like this as much as "Version 0.2x" I'd raise it with @omerbensaadon maybe since it was added to the word list that way. I don't care which we use as long as it's consistent.

Reply:
No problem, I will make the update.
I will go back and remove the "Version" from all.

Reply:
@abrennan89 I remove "Version " from all titles and linktitles so all are consistent now